### PR TITLE
ENGDOCS-2001

### PR DESCRIPTION
**Pull request by aevesdocker**
Refreshes the 'Try Compose' page with some more recent Compose features.

Adds more color
Gets to the Compose bits quicker and condenses set up into step one as opposed to two steps
Adds more links. 

(Viewing this quickstart guide as a 'jumping off point' into other docs such as Compose watch, how it works etc. Will mostly likely need a more in-depth/involved/advanced tutorial later in the year)

## Reviews

- [x] Technical review
- [ ] Editorial review

### CODE REVIEW

**Review by glours — APPROVED** [review]
Few improvements to do but LGTM otherwise

File: content/compose/gettingstarted.md [hunk 1]
```diff
@@ -214,141 +217,136 @@ image pulled from the Docker Hub registry.
...
-```
+   $ docker compose ps
+
+          Name                      Command               State           Ports         
+   -------------------------------------------------------------------------------------
+   composetest_redis_1   docker-entrypoint.sh redis ...   Up      6379/tcp              
+   composetest_web_1     flask run                        Up      0.0.0.0:8000->5000/tcp
+   ```
+
+- See `docker compose --help` to see other available commands.
```
Comment (craig-osterhout) [inline content/compose/gettingstarted.md:338]: Nit for existing wording. It's not wrong, but the repetitive caught my eye. Maybe `use` or `run`. 
```suggestion
- Run `docker compose --help` to see other available commands.
```

File: content/compose/gettingstarted.md [hunk 2]
```diff
@@ -214,141 +217,136 @@ image pulled from the Docker Hub registry.
...
+
+- If you started Compose with `docker compose up -d`, stop your services once you've finished with them:
+
+   ```console
+   $ docker compose stop
+   ```
+
+- You can bring everything down, removing the containers entirely, with the `down` command.
```
Comment (craig-osterhout) [inline content/compose/gettingstarted.md:346]: Nit for existing wording. It's not wrong, but suggest the full command.
```suggestion
- You can bring everything down, removing the containers entirely, with the `docker compose down` command. 
```

**Review by craig-osterhout — APPROVED** [review]
LGTM.
A couple of subjective nits for some existing wording.

**Review by craig-osterhout — APPROVED** [review]

**Pull request merged**